### PR TITLE
Journals

### PR DESCRIPTION
**Pull request by jmcreasman**

### CODE REVIEW

**Comment by jeffworley** [comment]
Great topics and starting points!  I'm looking forward to seeing how you elaborate on both. :+1:

**Comment by karjac14** [comment]
Its seems you got a lot of take homes from the field trip. great observation there! :+1:

File: tutorial-week-4.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+To be honest I have a difficult time coming up with topics for tutorial based journal entries but this one popped into my head so I figured I’d roll with it. Right now this is mostly just a stream of thoughts that I’ll flesh out over the week so bear with me if it doesn't make as much sense at first.
```
Comment (kellymurray) [inline tutorial-week-4.md]: How about a tutorial for something technical? Like how to do something in CSS, or HTML, or JS?

**Comment by boshnivolo** [comment]
I like your Weekend Game Plan haha. I feel like I definitely need to structure my weekends better :+1:

File: tutorial-week-4.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+##How to make the most out of your weekend:
```
Comment (kellymurray) [inline tutorial-week-4.md:1]: Focus on a technical tutorial next week.

**Comment by ShayHall** [comment]
:+1: These look good. I think a reflection on the companies we visit is a good thing to write about.  I also like that you focused on how to relax in your tutorial.  While this is a bootcamp and we are meant to work hard, there is something to being able to take a little time to rest so you can make it through the next week.

**Comment by rosalsm** [comment]
I like the work space at Bronto too!!! :+1:

**Comment by boshnivolo** [comment]
:+1: great journal!

**Comment by ShayHall** [comment]
:+1: Still looking good.  All my previous comments are still valid.

File: tutorial-week-4.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+###Friday:
+This day is spent mostly just researching the topics I’ll need to know as well as catch up on topics I’m still a dim bulb on. I also make out an outline of what order I’ll be tackling the assignments and how much time I think I’ll need to give to them. Sometime about Friday afternoon/evening I’ll start on the homework and make sure I’ve at least touched on everything. While I don’t give myself as much freetime on this day I do try and give myself an hour or two at night to relax.
+###Saturday:
+This is where the bulk of my work load gets completed. About 8 or 9 in the morning I’ll go to a local starbucks and start working. If I’ve done my research and outline well then I don’t have too much trouble working through everything. About 2 or 3 in the afternoon I take about an hour break and try to head to a park to walk around and clear my head. Then it’s back to work! The only other break I take is for dinner (need that fuel). By this point about 75/80% of my homework is done.
+###Sunday:
```
Comment (ShayHall) [inline tutorial-week-4.md:9]: Sleeping in is not cheating!

**Comment by rosalsm** [comment]
I think you are very efficient and organize Jon :+1:

**Pull request merged**